index.ts

```diff
@@ -112,20 +118,28 @@ export function encodeRunestoneUnsafe(runestone: RunestoneSpec): Buffer {
   }));
 
   let etching: Option<Etching> = None;
+  let etchingCommitment: string | undefined = undefined;
   if (runestone.etching) {
     const etchingSpec = runestone.etching;
-
-    if (!etchingSpec.rune && etchingSpec.spacers?.length) {
-      throw Error('Spacers specified with no rune');
+    let hasSpacers = false;
+    for (const spacer of SPACERS) {
+      if (runestone.etching?.rune?.includes(spacer)) {
+        hasSpacers = true;
+        break;
+      }
     }
 
-    if (
-      etchingSpec.rune &&
-      etchingSpec.spacers?.length &&
-      Math.max(...etchingSpec.spacers)! >= etchingSpec.rune.length - 1
-    ) {
-      throw Error('Spacers specified out of bounds of rune');
```

Comment:
You don't need this cuz it is inferred that SpacedRunes does correctly testing to identify the spacers.
By simplifying to the user it simplifies to maintainers too